modules/ROOT/pages/fd-deploy-app-from-rm.adoc

```diff
@@ -1,15 +1,18 @@
 = Deploy a Mule Application from Runtime Manager
 
-It might be the case that you are not allowed to deploy an application directly from Flow Designer. Your organization might require someone with the proper authority to deploy applications from Runtime Manager. You can share with that person a copy of your project as a `.jar` file to deploy your application with.
+It might be the case that you are not allowed to deploy an application directly from Flow Designer.
+Your organization might require someone with the proper authority to deploy applications from Runtime Manager.
+You can share with that person a copy of your project as a `.jar` file to deploy your application with.
 
 == Procedure
 
 . In Flow Designer, follow these steps:
 .. Ensure that your project is free of errors.
-.. Click the download icon, which is to the right of the Undo and Redo arrows in the top-left corner of the screen.
-.. Select *Download Package*.
+.. Export your project by clicking the dots next to the Test button and select *Export*.
+.. Select *Export project sources* to export just your project source files, or
+*Export mule application* to export your entire Mule app, then click *Export*.
```

Comment:
@mcherem suggest we are more explicit and guide them to use this option when trying to share a file to be used for deployment in Runtime manager... "Select *Export mule application* (Selecting this option will produce a `.jar` file which will include all the connector credentials (@mcoaker to confirm) as well as all artifacts needed by Runtime Manager) 

something like that

Reply:
Rewritten to reflect comments.